src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,30 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
-
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Accordion } from "./Accordion";
+
+describe("Accordion", () => {
+  // const onClickMock = jest.fn();
+  const testHeaderText = "Accordion title";
+  const testChildren = <p>Accordion content</p>;
+
+  const renderComponent = () => {
+    const defaultProps = {
+      title: testHeaderText,
+    };
+
+    render(<Accordion {...defaultProps}>{testChildren}</Accordion>);
+  };
+  it("renders without error", () => {
+    renderComponent();
+    expect(screen).not.toBeNull();
+  });
+
+  it("triggers Click event when clicked", async () => {
```

Comment:
You could have written a better test description, what is actually happening, the content is shown when the accordion is clicked right

```suggestion
  it("shows the content when the accordion button is clicked", async () => {
```